fpp

```diff
@@ -25,6 +25,9 @@ PYTHONCMD="python"
 function doProgram {
   # process input from pipe and store as pickled file
   $PYTHONCMD "$BASEDIR/src/processInput.py" "$@"
+  # if it failed, just fail now and exit the script
+  # this works for the looping -ko case as well
+  if [[ $? != 0 ]]; then exit $?; fi
```

Comment:
Yeah i saw your comment on the other issue. `$?` _should_ work even tho we piping in input. I secretly suspect curses wrapper is maybe catching or rethrowing or something and the exit code is being lost..?